functions/UpdateBookingSlot-BookToISV/run.csx

```diff
@@ -1,4 +1,8 @@
 ﻿#load "..\Shared\httpUtils.csx"
+#load "..\Shared\CompleteAppointmentDTO.csx"
+#load "..\Shared\TechnicalResource.csx"
+#load "..\Shared\EncryptionUtils.csx"
+#load "..\Shared\isv.csx"
```

Comment:
** Which is interesting in itself, it didn’t work when I last tried it, it might now 😊

The breakdown is interesting in itself. We should have a think more about the correct patterns to use here.